x-pack/test/functional/apps/dashboard/reporting/download_csv.ts

```diff
@@ -67,11 +68,15 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
     describe('E-Commerce Data', () => {
       before(async () => {
         await esArchiver.load('x-pack/test/functional/es_archives/reporting/ecommerce');
-        await esArchiver.load('x-pack/test/functional/es_archives/reporting/ecommerce_kibana');
+        await kibanaServer.importExport.load(
+          'x-pack/test/functional/fixtures/kbn_archiver/reporting/ecommerce.json'
+        );
       });
       after(async () => {
         await esArchiver.unload('x-pack/test/functional/es_archives/reporting/ecommerce');
-        await esArchiver.unload('x-pack/test/functional/es_archives/reporting/ecommerce_kibana');
+        await kibanaServer.importExport.unload(
+          'x-pack/test/functional/fixtures/kbn_archiver/reporting/ecommerce.json'
+        );
```

Comment:
nit: not to copy paste json path, I suggest using a variable when there are 2+ occurrences in the file:
```
const ecommerceArchivePath='x-pack/test/functional/fixtures/kbn_archiver/reporting/ecommerce.json'
await kibanaServer.importExport.load(ecommerceArchivePath)
```

Reply:
This way it will be clear that we use the same one through out the test file.
As for naming... maybe `ecommerceSOPath` is better. Thoughts?

Reply:
DRY is best

Reply:
Did the change in the all the 7 test files here @dmlemeshko. Thank you and thanks @wayneseymour